From 7cd46365573adc9fc6f6d1f0207f1fe3c720b846 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Fri, 18 Dec 2015 11:06:05 -0500
Subject: [PATCH 0659/1110] drm/amd/dal: Guard ASIC specific functions in
 display_clock header

Guarding unguarded ASIC specific function headers in
display_clock_interface.h. These should probably be
moved to ASIC specific files.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
---
 drivers/gpu/drm/amd/dal/include/display_clock_interface.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/amd/dal/include/display_clock_interface.h b/drivers/gpu/drm/amd/dal/include/display_clock_interface.h
index 2f48b8a..f3a1cee 100644
--- a/drivers/gpu/drm/amd/dal/include/display_clock_interface.h
+++ b/drivers/gpu/drm/amd/dal/include/display_clock_interface.h
@@ -132,13 +132,17 @@ struct display_clock_state {
 
 struct display_clock;
 
+#if defined(CONFIG_DRM_AMD_DAL_DCE11_0)
 struct display_clock *dal_display_clock_dce110_create(
 	struct dc_context *ctx,
 	struct adapter_service *as);
+#endif
 
+#if defined(CONFIG_DRM_AMD_DAL_DCE8_0)
 struct display_clock *dal_display_clock_dce80_create(
 	struct dc_context *ctx,
 	struct adapter_service *as);
+#endif
 
 void dal_display_clock_destroy(struct display_clock **to_destroy);
 bool dal_display_clock_validate(
-- 
2.7.4

